### PR TITLE
Task 2: webpack config

### PR DESCRIPTION
**Pull request by bellamy9678**

### CODE REVIEW

File: webpack.config.dev.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HTMLLoader = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.tsx',
+  mode: 'development',
+  output: {
+    filename: "[name].js",
```
Comment (Vladlan) [inline webpack.config.dev.js:8]: Add hashing for js files to invalidate browser cash when you add changes to you app

File: webpack.config.dev.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    path: path.resolve(__dirname, 'build'),
+  },
+  module: {
+    rules: [
```
Comment (Vladlan) [inline webpack.config.dev.js:12]: Highly likely in your react project you will have css, please also tune up webpack for this

File: src/utils/en.json [hunk 3]
```diff
@@ -1,10 +1,7 @@
 {
   "title": "Hello World!",
```
Comment (Vladlan) [inline src/utils/en.json]: Usually translations stored in translations folder. Utils folder usually is used for helper (utility) functions.

**Review by Vladlan — CHANGES_REQUESTED** [review]

File: webpack.config.dev.js (hunk 2 above)
Comment (bellamy9678) [reply webpack.config.dev.js:12]: Seems like yes, but i suppose I can use chakra without plain css, just .t(j)s, I'll remove it at the end if I don't use it.

**Review by Vladlan — APPROVED** [review]

**Pull request merged**